.travis.yml

```diff
@@ -0,0 +1,24 @@
+dist: xenial
+language: python
+sudo: false
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/.ccache  # https://github.com/travis-ci/travis-ci/issues/5853
+matrix:
+  fast_finish: true
+  include:
+    - python: 3.6
+    - python: 3.7
+    - python: 3.8
+    - python: nightly
+  allow_failures:
+    - python: nightly
+
+install:
+  - pip install -r requirements.txt
+  - pip install -r requirements-dev.txt
```

Comment:
Maybe here we can run the script with `-h`? Such as:
```bash
python resource_health_check.py -h
```

Reply:
Good idea. I did it in `test.py` via `subprocess.check_call` so that all the tests are in one place.

Reply:
Sounds great! I like the idea.